package-lock.json

```diff
@@ -22,6 +22,7 @@
         "core-js": "^3.13.1",
```

Comment:
I accidentally comitted this file - Should I keep it if this is what npm install gives me?